site/config.yaml

```diff
@@ -12,9 +12,10 @@ params:
   hero:
     backgroundColor: med-blue
   versioning: true
-  latest: v1.8
+  latest: v1.9.0-rc.1
```

Comment:
Is it automatically gen?

It should be v1.9

Reply:
I've changed